tests/js/spec/components/userMisery.spec.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import {shallow} from 'sentry-test/enzyme';
+
+import ScoreBar from 'app/components/scoreBar';
+import UserMisery from 'app/components/userMisery';
+
+describe('UserMisery', function() {
+  beforeEach(function() {});
+
+  afterEach(function() {});
+
+  it('renders', function() {
+    const wrapper = shallow(
+      <UserMisery
+        bars={10}
+        barHeight={20}
+        miseryLimit={300}
+        miserableUsers={75}
+        totalUsers={100}
+      />
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('renders no bars when miserable users is zero', function() {
+    const wrapper = shallow(
+      <UserMisery
+        bars={10}
+        barHeight={20}
+        miseryLimit={300}
+        miserableUsers={0}
+        totalUsers={100}
+      />
+    );
+    expect(wrapper.find(ScoreBar).props().score).toEqual(0);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('renders one bar when miserable users are close to zero', function() {
+    const wrapper = shallow(
+      <UserMisery
+        bars={10}
+        barHeight={20}
+        miseryLimit={300}
+        miserableUsers={1}
+        totalUsers={100}
+      />
+    );
+    expect(wrapper.find(ScoreBar).props().score).toEqual(1);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Snapshots will only bring you misery in the future. Are we able to use selectors to check that the correct number of bars have the right attributes?

Reply:
> Snapshots will only bring you misery in the future.

Ironic considering the component name? 😆 

For the sake of the test above, I think score is sufficient as it's just testing the `score` logic essentially.

Reply:
Removed snapshots 👍